filer/templates/admin/filer/widgets/admin_folder.html

```diff
@@ -1,28 +1,43 @@
 {% load i18n filer_admin_tags staticfiles %}
 
 {% spaceless %}
-    {% trans "Folder" %}:
-    {% if object %}
-        <span id="{{ span_id }}">{{ object.pretty_logical_path }}</span>
-    {% else %}
-        <span id="{{ span_id }}">{% trans "none selected" %}</span>
-    {% endif %}
-    {% render_filer_lookup_button %}
-    <img id="{{ clear_id }}" src="{% static 'admin/img/icon_deletelink.gif' %}" width="10" height="10" alt="{% trans 'Clear' %}" title="{% trans 'Clear' %}"{% if not object %} style="display: none;"{% endif %}>
-    <br>
-    {{ hidden_input }}
+    <div class="dropzone dropzone-folder">
+        <span class="filerFile">
+            <span id="{{ span_id }}">
+                {% trans "Folder" %}:
+                {% if object %}
+                    {{ object.pretty_logical_path }}
+                {% else %}
+                    {% trans "none selected" %}
+                {% endif %}
+            </span>
+            <a href="{{ lookup_url }}" class="related-lookup{% if object %} hidden{% endif %}" id="{{ lookup_name }}" title="{% trans 'Lookup' %}"{% if not IS_DJANGO_18 %} onclick="return showRelatedObjectLookupPopup(this);"{% endif %}>
+                {% trans 'Add Folder' %}
```

Comment:
even though you mention that the file does not clear our standards, still adapt touched code (use double quotes here)